cms/djangoapps/contentstore/management/commands/backfill_orgs_and_org_courses.py

```diff
@@ -121,6 +182,55 @@ def confirm_changes(options, orgs, org_courseid_pairs):
     return answer.lower().startswith('y')
 
 
+def bulk_add_data(orgs, org_courseid_pairs, dry_run, activate):
+    """
+    Bulk-add the organizations and organization-course linkages.
+
+    Print out list of organizations and organization-course linkages,
+    one per line. We distinguish between records that are added by
+    being created vs. those that are being added by just reactivating an
+    existing record.
+
+    Arguments:
+        options (dict[str]): command-line arguments.
```

Comment:
Leftover from an earlier version (no `options` being passed in)?

Reply:
Good catch. Fixed.